onnx/defs/math/defs.cc

```diff
@@ -1691,15 +2298,87 @@ ONNX_OPERATOR_SET_SCHEMA(
     10,
     OpSchema()
         .SetDoc(QLinearMatMul_ver10_doc)
```

Comment:
Let's also make all its inputs and outputs non-differentiable because they are integer tensors. The differentiable of integer doesn't exist in math so I prefer to go with a conservative setting here.